echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/bearychat/CreateMessageResponse.groovy

```diff
@@ -0,0 +1,20 @@
+package com.netflix.spinnaker.echo.bearychat
+
+
+
+class CreateMessageResponse {
```

Comment:
Can you add comments to these classes explaining which spec they are implementing?

Reply:
My mistake. I change my code but forget to delete this class.